types/object-overrides/index.d.ts

```diff
@@ -16,6 +16,9 @@ declare global {
     entries<K extends string | number, V>(
       o: Partial<Record<K, V>> | Record<string, never>
     ): [`${K}`, V][]
+    fromEntries<K extends string | number, T = any>(
```

Comment:
Potentially breaking change, but as far as I understand, this should break build during type resolution, not at runtime

Reply:
And it should only make types tighter, not looser

Reply:
yeah I'm concerned about breaking changes to other ecosystem, other than go-next, can you make sure that they "work" as intended?

Reply:
Checked, artifact substat, rv, mainstat calculations are the same across this deployment and another